public/app/features/admin/partials/users.html

```diff
@@ -55,7 +55,9 @@
             </a>
           </td>
           <td class="text-right">
-            <span class="label label-tag" ng-class="{'muted': user.isDisabled}" ng-if="ctrl.getAuthModule(user) === 'ldap'">LDAP</span>
+            <span class="label label-tag" ng-class="{'auth-label-muted': user.isDisabled}" ng-style="user.authLabelStyle" ng-if="user.authLabel">
```

Comment:
No need to mute these when the user is disabled, we have the disabled label for that